client/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "bartop-ui",
+  "version": "0.0.1",
+  "description": "Web app for BarTop's UI.",
+  "dependencies": {
+    "react": "^16.0.0",
+    "react-dom": "^16.0.0",
+    "react-scripts": "1.0.14"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test --env=jsdom",
+    "eject": "react-scripts eject"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/dpopp07/bartop.git"
+  },
+  "keywords": [
+    "bartop"
+  ],
+  "author": "Dustin Popp",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/dpopp07/bartop/issues"
+  },
+  "homepage": "https://github.com/dpopp07/bartop#readme"
```

Comment:
Let's go ahead and add the proxy field in the client package so that network requests get routed to our API. This comes from the article pasted in the server issue (#5) and Ram's article.

As a root value in the package.json object,

`"proxy": "http://localhost:3001"`

Reply:
Ah good looks